ingest/de_integration--cell_type__ontology_label--cholinergic_neuron--study--wilcoxon.tsv

```diff
@@ -0,0 +1,188 @@
+	names	scores	logfoldchanges	pvals	pvals_adj	pct_nz_group	pct_nz_reference
```

Comment:
@michelle-vel This file (and others under `ingest` and `test`) is an output file that does should not go into the repo. Also, the `dotfiles` - .zshrc should be removed from this PR.